api/src/main/java/org/openmrs/module/dhisconnector/api/impl/DHISConnectorServiceImpl.java

```diff
@@ -884,7 +884,7 @@ public String importMappings (MultipartFile mappingBundle) throws IOException {
 	}
 
 	@Override
-	public String[] exportMappings(String[] mappings) throws IOException {
+	public String[] exportMappings(String[] mappings, boolean dontIncludeMetadata) throws IOException {
```

Comment:
how about something like `shouldIncludeMetadata` ?

Reply:
sure, that is better

Reply:
@jayasanka-sack I made the requested change, please review 🙂